plugins/CoreConsole/tests/System/expected/test_ArchiveCronTest_noOptions__Goals.get_month.xml

```diff
@@ -6,9 +6,9 @@
 		<revenue>160</revenue>
 		<conversion_rate>94.12%</conversion_rate>
 		<nb_conversions_new_visit>32</nb_conversions_new_visit>
-		<nb_visits_converted_new_visit>31</nb_visits_converted_new_visit>
+		<nb_visits_converted_new_visit>32</nb_visits_converted_new_visit>
 		<revenue_new_visit>160</revenue_new_visit>
-		<conversion_rate_new_visit>91.18%</conversion_rate_new_visit>
+		<conversion_rate_new_visit>94.12%</conversion_rate_new_visit>
```

Comment:
This change is caused by invalidating the `VisitsSummary` archive, before archiving the goal metrics. As the goal archiver is using data from like `nb_visits`, it was using outdated data before.